test/generate_v2_unit_test_certs.sh

```diff
@@ -0,0 +1,100 @@
+# This generates a bunch of certs for unit tests of validating the rules
+# sections 3.2.2 and 3.2.3 of the V2 CT RFC. They are intended for use
+# in tests only. Some of the output certs do not follow the rules in the RFC.
+# Test case numbering relates to spreadsheet. Not all cases may be tested so
+# there could be gaps in numbering.
+
+set -e
+
+TMP_REQ=/tmp/req.csr
```

Comment:
Worth using `$(mktemp)` in here? (or mktemp -d and then blowing the whole dir away afterwards?)

There are some other `/tmp/file` references too if you decide to do this.